x-pack/plugins/apm/public/tutorial/config_agent/agent_config_instructions.tsx

```diff
@@ -60,7 +60,11 @@ export function AgentConfigInstructions({
       />
       <EuiSpacer />
 
-      <EuiCodeBlock isCopyable language="bash" data-test-subj="commands">
+      <EuiCodeBlock
+        isCopyable
+        language={variantId === 'java' ? variantId : 'bash'}
```

Comment:
`bash` works pretty well for the other languages but for java it doesn't highlight the syntax at all.  

Not sure if it's worth checking it though